L1Trigger/L1THGCal/plugins/be_algorithms/SingleCellClusterAlgo.cc

```diff
@@ -12,14 +13,14 @@ class SingleCellClusterAlgo : public Algorithm<HGCalTriggerCellBestChoiceCodec>
 
         SingleCellClusterAlgo(const edm::ParameterSet& conf):
             Algorithm<HGCalTriggerCellBestChoiceCodec>(conf),
-            cluster_product_( new l1t::HGCalTriggerCellBxCollection ){}
-
+            cluster_product_( new l1t::HGCalTriggerCellBxCollection ),
+            calibration_(conf){}
```

Comment:
Can you encapsulate the calibration parameters in one parameter set inside the main parameter set?